config/pmd/ruleset.xml

```diff
@@ -66,7 +66,6 @@
     <exclude name="UncommentedEmptyMethodBody"/>
   </rule>
   <rule ref="category/java/errorprone.xml">
-    <exclude name="BeanMembersShouldSerialize"/>
```

Comment:
This can be replaced with <exclude name="NonSerializableClass"/>

Looks like they changed the name

Reply:
Done :)